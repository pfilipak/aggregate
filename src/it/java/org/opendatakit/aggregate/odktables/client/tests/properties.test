# Set up
createUser user1

# Test that we can create column properties
createSynchronizedTable user1 people
    name STRING false props {'key1':'value1'}
    age INTEGER false {'key2':'value2'}
    weight DECIMAL false {'key3':'value3'}

# Clean up
deleteSynchronizedTable user1 people
deleteUser user1