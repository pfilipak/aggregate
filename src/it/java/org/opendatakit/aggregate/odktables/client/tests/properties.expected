# Set up
createUser user1

# Test that we can create column properties
createSynchronizedTable user1 people

# Clean up
deleteSynchronizedTable user1 people
deleteUser user1